python_apps/pypo/pypo/liquidsoap_scripts/liquidsoap_prepare_terminate.py

```diff
@@ -14,6 +14,6 @@
     tn.read_all()
 
 except Exception, e:
-    print('Error loading config file: %s', e)
+    print 'Error loading config file: %s' % e
```

Comment:
You replaced a line that's forward compatible with Python 3-compatible with a line that's only compatible with python 2...